sdk/src/main/java/com/mapbox/maps/attribution/AttributionParser.kt

```diff
@@ -70,6 +71,24 @@ open class AttributionParser internal constructor(
     addAdditionalAttributions()
   }
 
+  /**
+   * Parse string literal to attribution with empty url.
+   */
+  private fun parseStringLiteralToAttributions(stringLiteralArray: MutableList<String>) {
```

Comment:
is indeed a mutable list needed here?

Reply:
additionally naming `stringLiteralArray` with word array is not aligned well with actual list type

Reply:
yeah, I will update that.